website/docs/tests/maester/MT.1007.md

```diff
@@ -15,6 +15,7 @@ Checks if the tenant has at least one conditional access policy requiring multif
 
 MFA for all users conditional access policy can be used to require MFA for all users in the tenant.
 
-## Learn more
-
-- [Conditional Access policy: Require MFA for all users](https://learn.microsoft.com/entra/identity/conditional-access/howto-conditional-access-policy-all-users-mfa)
+## Related links
+- [Entra admin center - Conditional Access | Policies](https://entra.microsoft.com/#view/Microsoft_AAD_ConditionalAccess/ConditionalAccessBlade/~/Overview/fromNav/)
+- [Entra admin center - Conditional Access | Policy templates](https://entra.microsoft.com/#view/Microsoft_AAD_ConditionalAccess/CaTemplates.ReactView)
+- [Apply Conditional Access policies to every app](https://learn.microsoft.com/entra/identity/conditional-access/howto-conditional-access-policy-all-users-mfa)
```

Comment:
The link text should read "Common Conditional Access policy: Require MFA for all users"

Reply:
The link text has now been corrected. I'm sorry about that @f-bader